src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arr = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el !== '');
+  const nweArr = arr.map((el) => el.split(':').map((item) => item.trim()));
```

Comment:
[CODE STYLE] - Variable `nweArr` seems to be a typo and is also not descriptive. Please use a more descriptive name to reflect the purpose of the variable. Additionally, [CODE KNOWLEDGE] - Don't need to make a copy before each operation. The `map` method does not mutate the original array.